lib/rollbax/item.ex

```diff
@@ -4,10 +4,15 @@ defmodule Rollbax.Item do
     %{"access_token" => token,
       "data" => %{
         "server" => %{
-          "host" => List.to_string(host)},
+          "host" => List.to_string(host)
+        },
         "environment" => envt,
-        "language"  => "Elixir",
-        "framework" => "OTP"}}
+        "language"  => language(),
+        "framework" => framework(),
```

Comment:
About "another naming for OS information" we can execute os-dependent system commands and grab suitable matched info.

And I am agree that OS/Erlang/OTP/Elixir info should be put to platform, but framework remain for specific integration (i.e. Phoenix)

Purpose of this small PR is add more context information into Rollbar events about runtime environment